.circleci/config.yml

```diff
@@ -0,0 +1,33 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/sd
+    docker:
+      - image: msheiny/pressfreedomci:latest
```

Comment:
Umm nope - nevermind see line 27 under `devops/playbooks/aws-ci-startup.yml` for reference to ^^ image. The image above is from the `Dockerfile` below..... Fudge on second thought, I should add that to this repo for tracking. It is merely used as a `jump box` image to lauch other docker images from.

```
FROM debian:stretch


ENV DOCKER_BUCKET get.docker.com
ENV DOCKER_VER 17.05.0-ce
ENV DOCKER_SHA256_x86_64 340e0b5a009ba70e1b644136b94d13824db0aeb52e09071410f35a95d94316d9

RUN apt-get update && \
    apt-get install -y flake8 make virtualenv ccontrol libpython2.7-dev \
            libffi-dev libssl-dev libyaml-dev python-pip curl git &&\
    apt-get clean

RUN curl -L -o /tmp/docker-${DOCKER_VER}.tgz https://get.docker.com/builds/Linux/x86_64/docker-${DOCKER_VER}.tgz; \
	echo "${DOCKER_SHA256_x86_64} /tmp/docker-${DOCKER_VER}.tgz" | sha256sum -c -; \
	cd /tmp && tar -xz -f /tmp/docker-${DOCKER_VER}.tgz; \
	mv /tmp/docker/* /usr/bin

CMD /bin/bash
```